CoreFoundation/Base.subproj/CFFileUtilities.c

```diff
@@ -1276,7 +1276,7 @@ CFStringRef _CFXDGCreateRuntimeDirectoryPath(void) {
 }
 
 
-CF_PRIVATE CFArrayRef _CFTokenizeStringToCFArrayOfCFStrings(const char *values, char delimiter) {
+CF_PRIVATE CFArrayRef _CFCreateCFArrayByTokenizingString(const char *values, char delimiter) {
     size_t pathCount = 0;
     char* tmpDirectoriesPaths = (char*)values;
     char* last_colon = 0;
```

Comment:
@parkera  Sure, I would do the needed changes and update. 
When NULL/empty array is returned, one way of dealing with it is to construct the default values for the XDG variable and return. Does it sound ok to you?